packages/build/src/lib/schema-parser/middleware/addParameter.ts

```diff
@@ -20,15 +20,18 @@ export class AddParameter extends SchemaParserMiddleware {
     parameters.forEach((parameter) => {
       // We only check the first value of nested parameters
       const name = parameter.name.split('.')[0];
-      if (
-        !schemas.request!.some(
-          (paramInSchema) => paramInSchema.fieldName === name
-        )
-      ) {
+      const existedParam = schemas.request!.find(
+        (paramInSchema) => paramInSchema.fieldName === name
+      );
+
+      if (existedParam) {
+        if (!existedParam.validators) existedParam.validators = [];
+        existedParam.validators.push(...(parameter.validators || []));
```

Comment:
If there are the same name validators in metadata and schema requests, will we exclude one of them? 

Because we may face some cases which the args are different in schema request and SQL syntax validation filter:
- schema request has integer validator with args min=10, and max=20
- metadata exists in the validator with args min=5, and max=90 which parsed from SQL

Reply:
I pushed all validators to the list because I didn't know how to compose all of the validators, e.g.
string(format="\d+")
string(format="[1-5]+")

So I prefer to execute all of these validators.

P.S. Our constraints have composed logic so that we won't duplicate them.